mk/spksrc.cross-meson-env.mk

```diff
@@ -65,3 +69,53 @@ ifeq ($(findstring $(ARCH),$(x64_ARCHS)),$(ARCH))
   MESON_HOST_CPU = x86_64
   MESON_HOST_ENDIAN = little
 endif
+
+.PHONY: $(MESON_CROSS_TOOLCHAIN_PKG)
+$(MESON_CROSS_TOOLCHAIN_PKG):
+	@$(MSG) Generating $(MESON_TOOLCHAIN_PKG)
+	env $(MAKE) --no-print-directory meson_pkg_toolchain > $(MESON_CROSS_TOOLCHAIN_PKG) 2>/dev/null;
+
+.PHONY: meson_pkg_toolchain
+meson_pkg_toolchain: SHELL:=/bin/bash
+meson_pkg_toolchain:
+	@cat $(MESON_CROSS_TOOLCHAIN_WRK)
+	@echo
+	@echo "[properties]" ; \
+	echo "needs_exe_wrapper = false"
+ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS)),$(ARCH))
+	@echo "longdouble_format = 'IEEE_DOUBLE_BE'"
+else ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
+	@echo "longdouble_format = 'IEEE_DOUBLE_LE'"
+endif
```

Comment:
I suppose that all those archs have to use `IEEE_DOUBLE_LE` (i.e. liddle endian)
Only PPC_ARCHS have big endian byte ordering.

Reply:
thnx, an oversight of me part of my crusade.  Code is not ready for prime use just yet.